src/testIntegration/java/uk/gov/justice/digital/delius/controller/secure/OffendersResource_getOffenderCustodialStatusByCrnTest.java

```diff
@@ -0,0 +1,122 @@
+package uk.gov.justice.digital.delius.controller.secure;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.restassured.RestAssured;
+import io.restassured.config.ObjectMapperConfig;
+import io.restassured.config.RestAssuredConfig;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.justice.digital.delius.data.api.CustodialStatus;
+
+import java.time.LocalDate;
+
+import static io.restassured.RestAssured.given;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@ActiveProfiles("dev-seed")
+public class OffendersResource_getOffenderCustodialStatusByCrnTest {
```

Comment:
You can now extend `IntegrationTestBase` and get rid of a lot of the setup code, see all the other refactored integration tests